index.php

```diff
@@ -37,12 +37,20 @@ function __construct($config) {
     $this->contact = $config['contact'];
     $this->misc = $config['misc'];
     $this->routers = $config['routers'];
+    $this->doc = $config['doc'];
+  }
+
+  private function router_count() {
+    if ($this->frontpage['router_count'] > 0)
+      return $this->frontpage['router_count'];
+    else
```

Comment:
Maybe need add this:
```
elseif  ($this->frontpage['router_count'] > 0 AND $this->frontpage['router_count'] <= 5 }
 return count($this->routers);
esle 
return 1;
```
If have many router for like this http://lg.triolan.net.ua, - page will be long.

Reply:
Need fink about thith code